src/PowerShellEditorServices/Language/CommandHelpers.cs

```diff
@@ -57,13 +51,12 @@ public class CommandHelpers
                  commandInfo.CommandType == CommandTypes.Function ||
                  commandInfo.CommandType == CommandTypes.Filter))
             {
-                using (PowerShell powerShell = PowerShell.Create())
-                {
-                    powerShell.Runspace = runspace;
-                    powerShell.AddCommand("Get-Help");
-                    powerShell.AddArgument(commandInfo);
-                    helpObject = powerShell.Invoke<PSObject>().FirstOrDefault();
-                }
+                PSCommand command = new PSCommand();
+                command.AddCommand("Get-Help");
+                command.AddArgument(commandInfo);
```

Comment:
BTW over time should we nail these commands down to the ones that ship with PowerShell e.g. `Microsoft.PowerShell.Core\Get-Help`?  Now that the extension runs profiles, users could be pulling in modules that proxy commands like Get-Help.  Heck, PSCX does this.  This (our proxy) has caused some end user problems and we've had to back out some features.